From 0f70847107b1bd07b5b5de1bd105fb7e72713667 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@ew.tq-group.com>
Date: Tue, 22 Nov 2022 07:58:31 +0100
Subject: [PATCH] iMX8DXL: remove unused targets

u-boot-atf.itb is not used but references unavailable ./mkimage_uboot

Signed-off-by: Markus Niebel <Markus.Niebel@ew.tq-group.com>
---
 iMX8DXL/soc.mak | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/iMX8DXL/soc.mak b/iMX8DXL/soc.mak
index 6542712..47cf426 100755
--- a/iMX8DXL/soc.mak
+++ b/iMX8DXL/soc.mak
@@ -41,13 +41,6 @@ u-boot-atf.bin: u-boot-hash.bin bl31.bin
 	@cp bl31.bin u-boot-atf.bin
 	@dd if=u-boot-hash.bin of=u-boot-atf.bin bs=1K seek=128
 
-u-boot-atf.itb: u-boot-hash.bin bl31.bin
-	./$(PAD_IMAGE) bl31.bin
-	./$(PAD_IMAGE) u-boot-hash.bin
-	TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ./mkimage_fit_atf.sh > u-boot.its;
-	./mkimage_uboot -E -p 0x3000 -f u-boot.its u-boot-atf.itb;
-	@rm -f u-boot.its
-
 u-boot-atf-container.img: bl31.bin u-boot-hash.bin
 	if [ -f tee.bin ]; then \
 		if [ $(shell echo $(ROLLBACK_INDEX_IN_CONTAINER)) ]; then \
-- 
2.25.1

